UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -924,7 +926,7 @@ <h2>Examples</h2>
 				</section>
 
 				<section id="add-info-techniques">
-					<h4>Additional accessibility information metadata techniques</h4>
+					<h4>Display techniques for additional accessibility information support</h4>
```

Comment:
This heading also sounds confusing with "support" added to it. I would make it "Display techniques for additional accessibility information".